nncf/quantization/algorithms/min_max/backend.py

```diff
@@ -148,13 +148,23 @@ def get_weight_tensor(model: TModel, node: NNCFNode) -> Tuple[str, np.ndarray]:
         """
 
     @staticmethod
-    def get_weight_tensor_port_id(model: TModel, node: NNCFNode) -> Optional[int]:
+    def get_weight_tensor_port_id(model: TModel, node: NNCFNode) -> int:
```

Comment:
`get_weight_tensor_port_id` here and now `get_weight_tensor_port_id` in the `ONNXGraph`?

Reply:
Done. Changed `get_weight_tensor_port_id `in `ONNXGraph` to `get_weight_port_id `